<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HillTopWidgetForm</class>
 <widget class="QWidget" name="HillTopWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hill Top</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Hill Top</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>Input Parameters</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Layer with raster DEM</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QComboBox" name="m_layersComboBox"/>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Layer with raster Slope</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_slopeLayerComboBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="2" column="0">
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Layer with vector Contour Lines</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_8">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_contourLayerComboBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Layer with vector Drainage</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_10">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_drainageLayerComboBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="4" column="0">
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Layer with vector Watersheds</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_12">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_watershedsLayerComboBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="5" column="0">
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Layer with vector Watersheds Buffer</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_14">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_watershedsBufferLayerComboBox"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="6" column="0">
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Height (m)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="m_heightLineEdit">
                   <property name="text">
                    <string>50</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="7" column="0">
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Slope</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="m_slopeLineEdit">
                   <property name="text">
                    <string>17</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>               
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>Output Parameters</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QToolButton" name="m_fileToolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="m_fileLineEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
